File: June-18/24-06-18/java/01-Roy-and-profile-picture.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int l = input.nextInt();
        int n = input.nextInt();
        while(n-- > 0)
        {
            int w, h;
            w = input.nextInt();
            h = input.nextInt();
            if(w < l || h < l)
                System.out.print("UPLOAD ANOTHER\n");
            else
            {
                if(w == h)
                    System.out.print("ACCEPTED\n");
                else
                    System.out.print("CROP IT\n");
            }
            
        }
        
        
    }
}

File: June-18/24-06-18/python/01-Roy-and-profile-picture.py
l = int(input())

for _ in range(int(input())):
    w, h = map(int, input().split())
    if(w < l or h < l):
        print("UPLOAD ANOTHER")
    else:
        if(w == h):
            print("ACCEPTED")
        else:
            print("CROP IT")
            
